AudioKit/Common/Nodes/Playback/Player/AKAudioPlayer.swift

```diff
@@ -236,6 +236,7 @@ open class AKAudioPlayer: AKNode, AKToggleable {
     }
     
     deinit {
+        AKLog("* AKAudioPlayer")
```

Comment:
Interestingly when AKLog actually outputs that it shows up as:
deinit * AKAudioPlayer

That's pretty clear.